packages/frontend/src/utils/encryption/index.ts

```diff
@@ -21,3 +22,25 @@ export const encryptWallet = async (password: string): Promise<void> => {
         }
     }
 };
+
+export const decryptWalletKeys = async (password: string): Promise<void> => {
+    const encryptedkeyStore = new nearApiJs.keyStores.BrowserLocalStorageKeyStore(
+        new EncrytedLocalStorage(createKeyFrom(password)),
+        KEYSTORE_PREFIX
+    );
+
+    const decryptedkeyStore = new nearApiJs.keyStores.BrowserLocalStorageKeyStore(
+        window.localStorage,
+        KEYSTORE_PREFIX
+    );
+
+    const allAccounts = wallet.getAccountsLocalStorage();
+
+    for (let accountId in allAccounts) {
+        const keyPair = await encryptedkeyStore.getKey(CONFIG.NETWORK_ID, accountId) as KeyPairEd25519;
+        await encryptedkeyStore.removeKey(CONFIG.NETWORK_ID, accountId);
+        await decryptedkeyStore.setKey(CONFIG.NETWORK_ID, accountId, keyPair);
+    }
+
+    wallet.injectKeyStore(decryptedkeyStore);
```

Comment:
In the encryption flow we have a separate method `injectEncryptedKeyStore`. Won't it be more expected to move part of this code into `injectDecryptedKeyStore`?